site/oauth2-examples-proxy.md

```diff
@@ -0,0 +1,75 @@
+# Use OAuth2 Proxy and Keycloak as OAuth 2.0 server
+
+Let's test the following flow:
+
+* Access management ui via a browser thru OAuth2-Proxy
+
+<pre class="lang-plain">
+                    [ Keycloak ] 3. authenticate
+                      /|\  |
+                       |   | 4. token
+        2.redirect     |  \|/                                        [ RabbitMQ ]
+                [ Oauth2-Proxy ]       ----5. forward with token-->  [  http    ]
+                      /|\
+                       |
+            1. rabbit_admin from a browser
+</pre>
+
+## Prerequisites to follow this guide
+
```

Comment:
Should the heading here read "Prerequisites to use OAuth2 Proxy", it isn't really a guide it is a topic with procedures to setup OAuth2 Proxy, also I think updating the heading will help with SEO (retrieving this topic in google searches)

Reply:
Hi @pstack2021 , the prerequisites are only to follow the instructions shown on this use case or guide on how to use OAuth2 Proxy and Keycloak with RabbitMQ. Not necessarily to use OAuth2 proxy. We have 3 more guides like this one where we use the same heading. 
Will the main heading, h1, be enough to help with SEO? That was a very nice advice !